src/applications/caregivers/utils/helpers/form-config.js

```diff
@@ -0,0 +1,34 @@
+import recordEvent from 'platform/monitoring/record-event';
+import { MED_CENTERS_BY_STATE } from '../constants';
+
+export const createPayload = (file, formId, password) => {
+  const payload = new FormData();
+  payload.append('attachment[file_data]', file);
+  payload.append('form_id', formId);
+  payload.append('name', file.name);
+
+  // password for encrypted PDFs
+  if (password) {
+    payload.append('attachment[password]', password);
+  }
+
+  return payload;
+};
+
+export const parseResponse = (fileInfo, file) => {
+  recordEvent({
+    'caregivers-poa-document-guid': fileInfo.data.attributes.guid,
+    'caregivers-poa-document-confirmation-code': fileInfo.data.id,
+  });
+
+  return {
+    guid: fileInfo.data.attributes.guid,
+    confirmationCode: fileInfo.data.id,
+    name: file.name,
+  };
+};
+
+export const setPlannedClinics = formData => {
+  const state = formData['view:plannedClinicState'];
+  return state ? { enum: MED_CENTERS_BY_STATE[state] || [] } : { enum: [] };
```

Comment:
I think this can be shortened to `  return { enum: MED_CENTERS_BY_STATE[state] || [] };` since if state is empty it returns the empty array. It passes all the existing tests too. Although that is changing the implementation here so it's fine to ignore this too 🤷 .